www-apps/icingaweb2-module-director/icingaweb2-module-director-9999.ebuild

```diff
@@ -1,38 +1,50 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
-DESCRIPTION="Icinga Web 2 plugin for configuration"
-HOMEPAGE="https://dev.icinga.org/projects/icingaweb2-module-director/"
 inherit systemd
+
+DESCRIPTION="Icinga Web 2 plugin for configuration"
+HOMEPAGE="https://github.com/Icinga/icingaweb2-module-director/"
 if [[ "${PV}" == 9999 ]]; then
 	inherit git-r3
-	EGIT_REPO_URI="https://github.com/Icinga/icingaweb2-module-director.git"
+	EGIT_REPO_URI="https://github.com/Icinga/${PN}.git"
 else
 	KEYWORDS="~amd64 ~x86"
-	MY_PN="icingaweb2-module-director"
-	SRC_URI="https://codeload.github.com/Icinga/${MY_PN}/tar.gz/v${PV} -> ${P}.tar.gz"
-	S="${WORKDIR}/${MY_PN}-${PV}"
+	SRC_URI="https://codeload.github.com/Icinga/${PN}/tar.gz/v${PV} -> ${P}.tar.gz"
+	S="${WORKDIR}/${PN}-${PV}"
 fi
 
 LICENSE="GPL-2"
 SLOT="0"
 
-DEPEND=">=net-analyzer/icinga2-2.6.0
-	>=www-apps/icingaweb2-2.6.0
-	|| (
-		dev-lang/php:7.3[curl,iconv,pcntl,posix,sockets]
-		dev-lang/php:7.4[curl,iconv,pcntl,posix,sockets]
-		dev-lang/php:8.0[curl,iconv,pcntl,posix,sockets]
-	)
+# Dependencies according to https://github.com/Icinga/icingaweb2-module-director/blob/master/doc/02-Installation.md
+IUSE="php_targets_php8-1"
+PHP_DEPEND="
+	php_targets_php8-1? ( dev-lang/php:8.1[curl,iconv,pcntl,posix,sockets] )
```

Comment:
same edits here too

Reply:
I really would like to, but icingaweb2-module-director is not php 8.2 compatible yet